code/package.json

```diff
@@ -4,12 +4,13 @@
     "workspaces/*"
   ],
   "scripts": {
-    "update-version": "shx echo \"{ \\\"version\\\": \\\"`git describe --tags --always`\\\" }\" > version.json"
+    "update-version": "shx echo \"{ \\\"version\\\": \\\"`git describe --tags --always`\\\" }\" > version.json",
+    "check-not-master": "git branch | egrep \"\\*\"  | egrep -v \"^\\* master$\""
```

Comment:
Could this be simplified to `git branch | egrep '^\* master$'`?

Reply:
No, "For a commit to be blocked, pre-commit script must exit with a non-zero exit code." (https://www.npmjs.com/package/husky).
```
$ git branch | egrep '^\* master$'
* master
$ echo $?
0
$ git branch | egrep "\*"  | egrep -v "^\* master$"
$ echo $?
1
```

Reply:
Ahhh I see, thanks for the explanation!